==> creating model 'hg_mbn', stacks=2, blocks=1
=> no checkpoint found at 'checkpoint/mpii/hg_mbn_s2_b1_e2/model_best.pth.tar'
    Total params: 8.20M
    Mean: 0.4404, 0.4440, 0.4327
    Std:  0.2458, 0.2410, 0.2468

Epoch: 1 | LR: 0.00025000
Traceback (most recent call last):
  File "example/main.py", line 434, in <module>
    main(parser.parse_args())
  File "example/main.py", line 158, in main
    njoints, args.debug, args.flip)
  File "example/main.py", line 272, in validate
    for i, (input, target, meta) in enumerate(val_loader):
  File "/usr/local/python3.7/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    idx, batch = self._get_batch()
  File "/usr/local/python3.7/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 601, in _get_batch
    return self.data_queue.get(timeout=MP_STATUS_CHECK_INTERVAL)
  File "/usr/local/python3.7/lib/python3.7/queue.py", line 179, in get
    self.not_empty.wait(remaining)
  File "/usr/local/python3.7/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
  File "/usr/local/python3.7/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 274, in handler
    _error_if_any_worker_fails()
RuntimeError: DataLoader worker (pid 9387) is killed by signal: Killed. 
